x-pack/plugin/mapper-constant-keyword/src/test/java/org/elasticsearch/xpack/constantkeyword/mapper/ConstantKeywordFieldMapperTests.java

```diff
@@ -127,4 +130,27 @@ public void testMeta() throws Exception {
                 new CompressedXContent(mapping3), MergeReason.MAPPING_UPDATE);
         assertEquals(mapping3, mapper.mappingSource().toString());
     }
+
+    public void testLookupValues() throws Exception {
+        IndexService indexService = createIndex("test");
+        String mapping = Strings.toString(XContentFactory.jsonBuilder().startObject().startObject("_doc")
+            .startObject("properties").startObject("field").field("type", "constant_keyword")
```

Comment:
Really minor: all the indentations for the lines containing method calls in `testLookupValues` are inconsistent compared to the other methods in this class.

Reply:
👍